common/app/views/fragments/javaScriptLaterSteps.scala.html

```diff
@@ -14,24 +14,25 @@
             @curlPaths.map { case (module, path) =>
                 '@module': '@Static(path)',
             }
-            core:                      '@Static("javascripts/core.js")',
```

Comment:
'bootstraps/image-content': is the new longest line so I had to add one space to all lines :smile: